server/database/queries/provider/getOrderRequestQuery.js

```diff
@@ -0,0 +1,12 @@
+const connection = require('../../config/connection');
+
+const getOrderRequestQuery = (providerId) => {
+  const sql = {
+    text:
+      'SELECT description, state, date FROM ordes_request WHERE provider_id = $1;',
```

Comment:
add where status = ' pending' to query, here the provider can accept or delete the order request you just need the pending state